alchemiscale/storage/statestore.py

```diff
@@ -319,6 +320,11 @@ def _gufe_to_subgraph(
                 ):
                     node[key] = json.dumps(value, cls=JSON_HANDLER.encoder)
                     node["_json_props"].append(key)
+            elif isinstance(value, Settings):
+                # TODO: finish up approach here for serializing settings
```

Comment:
Is this still TODO?

Reply:
Nope! All accounted for. Thanks for catching this!